cypress/integration/login.js

```diff
@@ -7,18 +7,22 @@ describe('Login functionality', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
         cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="Demo User"]').click()
+        cy.get('[aria-label="TestCon User 25"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.user-info__title').contains('TestCon User 25')
+        cy.get('.main-nav').find('li').should('have.length', 2)
+    })
+    
+    it('Shoud be selected today', function (){
+        cy.get('.calendar--selected').should('have.class','calendar--today')
```

Comment:
We should also assert the numeric value of the selected date to match that of today :)

Reply:
Thanks for comment. I updated test